app/config/settings.py

```diff
@@ -1071,6 +1071,7 @@ def strtobool(val) -> bool:
 WORKSTATIONS_OVERLAY_QUERY_PARAM = "overlay"
 WORKSTATIONS_READY_STUDY_QUERY_PARAM = "readerStudy"
 WORKSTATIONS_ALGORITHM_JOB_QUERY_PARAM = "algorithmJob"
+WORKSTATIONS_ARCHIVE_ITEM_QUERY_PARAM = "archive-item"
```

Comment:
Change to camelCase to be in line with other parameters.

```suggestion
WORKSTATIONS_ARCHIVE_ITEM_QUERY_PARAM = "archiveItem"
```